src/states/DeckBuildingState.py

```diff
@@ -20,8 +21,8 @@ def __init__(self):
         self.selectedCardSpacing = 20
         self.availableCardWindow = 0
         self.isMouseOn = False
-        self.cardClass = CardClass #[CardClass.COMMON, CardClass.WARRIOR, CardClass.RANGER, CardClass.MAGE]
-        self.cardEffect = [EffectType.ATTACK, EffectType.MOVE, EffectType.SELF_BUFF, EffectType.RANGE_BUFF, EffectType.PULL, EffectType.PUSH, EffectType.DEBUFF, EffectType.BUFF, EffectType.CLEANSE, EffectType.HEAL, EffectType.DISCARD]
+        self.cardClass = list(CardClass)
+        self.cardEffect = list(EffectType)
```

Comment:
This would include all the effect type. And that is a little too much in my opinion. That is why I set it to a few of them. Look at the deck building filter and you know what I am talking about ahhaha.

Reply:
Yep it has too many effects. In my opinion, I think it will be better when debugging since we can filter the card with any effect. But if you think this case won't happen that much, which effects should I use for the filtering?

Reply:
The same as before should be fine

Reply:
ok